server/channels/api4/license.go

```diff
@@ -239,24 +244,32 @@ func requestTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 }
 
 func getPrevTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
+	// Check if the LicenseManager is initialized
```

Comment:
While the comment is not wrong, it also unrelated to the purpose of the PR. Can you please remove it and the other comments you added?

Reply:
Done. Thank you!